jupyter_telemetry/eventlog.py

```diff
@@ -66,6 +67,16 @@ class EventLog(Configurable):
         """
     ).tag(config=True)
 
+    json_validator = CaselessStrEnum(
+        JSON_SCHEMA_VALIDATORS.keys(),
+        'fastjsonschema',
+        help="""
+        JSON schema validation implementation.
+
+        Valid choices are `jsonschema` and `fastjsonschema`.
+        """
+    ).tag(allowed_none=False)
```

Comment:
@Zsailer Should we allow users to subclass here? They might want to customize the default validator or use their own. Or we can compromise and allow them to past a class name but restricted to only `JSONSchemaValidator` and `FastJSONSchemaValidator` and later can consider passing any class if someone asks about that use case. Is there any downside to allowing subclassing?

Reply:
Good question. I don’t see any downside to allowing custom validators by subclassing `EventSchemaValidator`. We can make this trait a `Type(default_value='FastJSONSchemaValidator', klass='EventSchemaValidator')`.